README.md

```diff
@@ -73,41 +74,247 @@ function MyComponent() {
 }
 ```
 
-## TOC
+# Table of Contents
 
-- APIs
+- API
   - [GraphQLClient](#GraphQLClient)
   - [ClientContext](#ClientContext)
-  - [useClient](#useClient)
   - [useQuery](#useQuery)
   - [useManualQuery](#useManualQuery)
   - [useMutation](#useMutation)
 - Guides
   - [SSR](#SSR)
   - [Authentication](#Authentication)
-  - [Refetching a query](#Refetching-a-query)
   - [Fragments](#Fragments)
+  - [Migrating from Apollo](#Migrating-from-Apollo)
 
 ## API
 
-### `GraphQLClient`
+## `GraphQLClient`
 
-### `ClientContext`
+**Usage**:
 
-### `useClient`
+```js
+import { GraphQLClient } from 'graphql-hooks';
+const client = new GraphQLClient(config);
+```
+
+**`config`**: Object with containing configuration properites
+
+- `url` (**Required**): The url to your GraphQL server
+- `ssrMode`: Boolean - set to `true` when using on the server for server-side rendering; defaults to `false`
+- `cache`: Object with the following methods:
+  - `cache.get(key)`
+  - `cache.set(key, data)`
+  - `cache.delete(key)`
+  - `cache.clear()`
+  - `cache.keys()`
+  - `getInitialState()`
+  - See [graphql-hooks-memcache](https://github.com/nearform/graphql-hooks-memcache) as a refernce implementation
```

Comment:
Typo
```suggestion
  - See [graphql-hooks-memcache](https://github.com/nearform/graphql-hooks-memcache) as a reference implementation
```

Maybe worth noting that it's not just a reference, it can be installed & used directly?